docs/SchemaMigrationGuide.md

```diff
@@ -1,53 +1,54 @@
 # Schema Migration Guide
-This document describes how to perform Sql schema migration on the application server.
+As Microsoft continues to support the newer releases of FHIR, migrations for the FHIR Server for Azure SQL Server edition will be required.
```

Comment:
It might make sense to move the description of what a migration is a bit further up in the documentation, before you start using the word "migration".

Maybe something like this:

`As Microsoft continues to develop the FHIR Server for Azure SQL Server and support new releases of FHIR, updates will need to be made to the T-SQL script that alters the SQL database. 
This document describes how to perform an update to the T-SQL script on the FHIR Server for Azure application server. This is referred to as a SQL Server schema migration.`